community/kernel/src/main/java/org/neo4j/kernel/impl/util/collection/TimedRepository.java

```diff
@@ -203,6 +193,12 @@ public void release( KEY key )
         return repo.keySet();
     }
 
+    public VALUE getValue( KEY key )
+    {
+        Entry entry = repo.get( key );
+        return entry == null ? null : entry.value;
+    }
+
```

Comment:
agree to some extent, `TimedRepostiory` itself participating in a test so replacing is not what is needed for this test, but i hide it to be protected instead of public method.